nucliadb_ingest/nucliadb_ingest/processing.py

```diff
@@ -216,6 +216,28 @@ async def convert_filefield_to_str(self, file: models.FileField) -> str:
             jwttoken = await resp.text()
         return jwttoken
 
+    def convert_external_filefield_to_str(
+        self, file_field: models.ExternalFileField
+    ) -> str:
+        if self.nuclia_jwt_key is None:
+            raise AttributeError()
+
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
+        expiration = now + datetime.timedelta(days=self.days_to_keep)
+        payload = {
+            "iss": "urn:nucliadb",
+            "sub": "file",
+            "aud": "urn:proxy",
+            "iat": now,
+            "exp": expiration,
+            "jti": uuid.uuid4().hex,
+            "source": 1,  # To indicate that this files comes internally
+            "driver": models.CloudFile.Source.EXTERNAL.value,
```

Comment:
If i read this correctly, this would resolve to "EXTERNAL", where in the other token payload self.driver is an `int`. This values needs to match the stashify_protos.processing.Driver protobuffer, that is not a dependency of nucliadb, but its is in terms of what this values needs to be.

I think there is a bit of naming problem/overlapping between the concept of source and driver that makes this a little diffucult to follow sometimes.

Reply:
BTW, the value needs to be `3`